ramls/loginCredentials.json

```diff
@@ -15,7 +15,7 @@
       "description": "User password",
       "type": "string"
     },
-    "tenantId": {
+    "tenant": {
       "description": "Tenant id; can be provided to override the current tenant",
```

Comment:
Can you add more detail to this description? What does "overriding the current tenant" do exactly in the context of this module? Probably don't need the bit before the `;`.

Reply:
Updated the description. Also if you want to know more deeply about a functionality the tenant field please take a look to [PR](https://github.com/folio-org/ui-consortia-settings/pull/23). Thank you